src/index.js

```diff
@@ -0,0 +1,60 @@
+import './style.css';
+
+const tasksList = document.getElementById('tasks-list');
+
+class Task {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
+  }
+
+  static tasks = [];
+
+  updateList() {
+    this.foo = 'eslint use this';
+    tasksList.innerHTML = '';
+    Task.tasks.forEach((task) => {
```

Comment:
- [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.id - b.id)`

Reply:
@brytebee Thank you for your comment!
I have worked on that!